### DISCUSSION TITLE
Reverse FrontText and BackText button.

### DISCUSSION THREAD

**Discussion by Joom25** [sviatoslav-lebediev/anki-quizlet-importer-extended#217] [Ideas]
I facing a problem. When some quizlet deck isn't following order. I have to manually reverse it, by changing the card front text and back text. It would be very helpful to just have a button to do that. Thank youi

**Comment by sviatoslav-lebediev** [comment]
Hi, sorry I didn't see this `discussion`, it should be an issue :) Let's continue here https://github.com/sviatoslav-lebediev/anki-quizlet-importer-extended/issues/235
